src/renderer/views/About/About.scss

```diff
@@ -13,8 +13,29 @@
 }
 
 .logo {
-  max-inline-size: 100%;
-  inline-size: 500px;
+  block-size: 200px;
+  inline-size: 640px;
+  display: inline-flex;
+  background-image: url('../../../../_icons/logoWhite.png');
+
+  @at-root {
+    .dracula &,
+    .CatppuccinMocha &,
+    .hotPink & {
+      background-image: url('../../../../_icons/logoWhite.png');
```

Comment:
I see that you're using mapping to the these three logo icons because we don't have full logo images for all of our themes. This is probably the best easily available solution, but it may need adjustment to meet FreeTube's muster. 

I'll talk with the FreeTube team about what we can do next. In the meantime, thank you for your initiative in working on this, and I'll see what we can do to work this out.

Reply:
As far as I can see, you have two choices:
1. I used an online tool to vectorize our existing logo. You can add it to `_icons`, add it inline to the template, and use the `fill` CSS property to dynamically change the color to the `--primary-text-color` for that theme.
![logoBlack](https://github.com/FreeTubeApp/FreeTube/assets/84899178/9591151d-8504-4f76-98b9-033fed5d31d8)
2. You can use an editor like GIMP to create logo images for themes missing one (with the fill color of that theme's `--primary-text-color`).

Reply:
I am little confused, what exactly do i need to do?

Reply:
Essentially, we need to have the appropriate theme color logo for each theme instead of mapping to the white, black, and default logos. These are the three ways that that can be achieved, and any of them should work. I apologize that this is a bit harder of an issue than was originally thought.

Reply:
okay got it, clear now. I would like to continue working on this. I will update this MR in couple of days with the listed solution.

Reply:
Hi @Shubh8899 just checking in, are u running in any issues with this?